thrift/jaeger.thrift

```diff
@@ -58,7 +58,7 @@ struct Span {
   4:  required i64           parentSpanId # since nearly all spans will have parents spans, CHILD_OF refs do not have to be explicit
   5:  required string        operationName
   6:  optional list<SpanRef> references   # causal references to other spans
-  7:  required i32           flags        # tbd
+  7:  required i32           flags        # a bit field used to propagate sampling decisions. 1 signifies a SAMPLED span, 2 signifies a DEBUG span.
```

Comment:
the field has to remain `i32` to avoid a breaking change, so I don't think enums are useful.